### ISSUE TITLE
中断训练过程后再继续训练

### ISSUE THREAD

**Issue by woshizhangcheng** [Stinky-Tofu/YOLO_V3#83]
您好， 训练过程中中断后再接着训练的话，会出现重头开始训练的效果， 是不是中断继续训练加载的模型参数有问题？

**Comment by Stinky-Tofu** [comment]
目前只支持重头训练

**Comment by woshizhangcheng** [comment]
再次麻烦您了， 由于标注的label尺寸大概10*15的大小， 检测较差，尝试修改config.py文件中的参数STRIDES = [4,8,16]后，对于loss_lbbox求损失会出现维度不匹配，如下错误在求sigmod交叉熵： [32,13,13,3,1]  vs   [32,26,26,3,1] 
respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
其中26是416/16, 但13*13怎么没有变化，是不是代码写死了，我暂时没找到这个问题点
而前面两个小的bbox没问题均能匹配
谢谢 @Stinky-Tofu

**Comment by lsk46029892** [comment]
你好，这个是可以断点续训的，将train.py文件的第85行注释掉，这个存储的模型是之前mobilenet的参数，将第100行改为“self.__save.restore(self.__sess, self.__weights_init)”，其中的 self.__weights_init的路径改为你中断之前保存的模型名称即可。我就是这样实现的，你可以参考下
